core/state/statedb_utils.go

```diff
@@ -153,34 +153,65 @@ func GetVoterCap(statedb *StateDB, candidate, voter common.Address) *big.Int {
 	return ret.Big()
 }
 
+func IncrementMintedRecordNonce(statedb *StateDB) {
+	nonce := statedb.GetNonce(common.MintedRecordAddressBinary)
+	statedb.SetNonce(common.MintedRecordAddressBinary, nonce+1)
+}
+
 var (
-	slotMintedRecordTotalMinted  uint64 = 0
-	slotMintedRecordLastEpochNum uint64 = 1
+	// Storage slot locations (32-byte keys) within MintedRecord SMC
+	slotMintedRecordOnsetEpoch             = common.HexToHash("0000000000000000000000000000000000000000000000000000000000000001")
+	slotMintedRecordOnsetBlock             = common.HexToHash("0000000000000000000000000000000000000000000000000000000000000002")
+	slotMintedRecordPostTotalMintedBase, _ = new(big.Int).SetString("0100000000000000000000000000000000000000000000000000000000000000", 16)
+	slotMintedRecordPostTotalBurnedBase, _ = new(big.Int).SetString("0200000000000000000000000000000000000000000000000000000000000000", 16)
+	slotMintedRecordPostRewardBlockBase, _ = new(big.Int).SetString("0300000000000000000000000000000000000000000000000000000000000000", 16)
```

Comment:
Can we add `0x` prefix here ? I think it's more clear

Reply:
`common.HexToHash` can. done.
`new(big.Int).SetString` cannot.